lib/stm32/h7/Makefile

```diff
@@ -46,7 +46,7 @@ OBJS += gpio_common_all.o gpio_common_f0234.o
 OBJS += pwr.o rcc.o
 OBJS += rcc_common_all.o
 OBJS += rng_common_v1.o
-OBJS += spi_common_all.o spi_common_v2.o
+OBJS += spi_common_v3.o
```

Comment:
spi_common_all.o should be retained so that spi_common_v3.o only implements v3 specific behaviour.